data/languages.json

```diff
@@ -122,6 +126,7 @@
   "ckb": {"nativeName": "کوردیی ناوەندی"},
   "clc": {},
   "co": {"nativeName": "corsu"},
+  "coa": {"nativeName": "Cocos Malay"},
```

Comment:
I think the intention is to use the native name rather than the English name in the `nativeName` property.

Reply:
I might be missing something, but I think the current override system only lets you define `nativeName`, you can't define translated names. So misusing `nativeName` is the only easy solution unless we want to rework this build script. 

If we do change the build logic, we need to decide if all translated labels should be stored in this same JS object, or if the overrides should be synced to transifex[^1]

[^1]: Even if we sync these overrides to transifex, the default language (english) would still need to be defined in this file along with `nativeName`.

Reply:
I would have said that the actual native name of the language would be preferred here, if known. 

We could in the future enhance the build scripts to include one or a few "fallback names" in other languages (e.g. English or a common language of the region) for cases when we don't know the language's native name or to simply make it more accessible to an international audience.

I wouldn't go as far as starting to translate those on transifex, because it would essentially duplicate the work done by the CLDR project, where it would be better to [contribute there](https://cldr.unicode.org/#how-to-contribute).

Reply:
Regarding native names vs english: I've updated all of them to use the native names.

1e980be0f440fe798056131544c16392ac3e8d27 is a proof-of-concept to allow translatable labels, which we could consider in a future PR